vendor/assets/javascripts/jquery.hammer.js

```diff
@@ -0,0 +1,37 @@
+// jQuery wrapper around HammerJS lib
+// Copyright Hammer.js licensed MIT
+// https://github.com/hammerjs/jquery.hammer.js
+
+(function(factory) {
+    if (typeof define === 'function' && define.amd) {
```

Comment:
The rails asset is compatible with jQuery 2 only, which is why I originally opened #6557. Until the codebase is migrated to jQuery 2, we have to keep this file. I will correct the linter remarks, though.